kubernetes/annotations_and_labels.md

```diff
@@ -15,10 +15,8 @@ This page defines common annotations and labels we set in Kubernetes objects.
 
 ## Common Labels
 
-- `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This together with `version`
-  label should also be used as replicas selector. E.g. `app=kvm-operator,version=1.0.0`. Exceptions can
-  be made to accomodate for adherence to existing selectors upstream.
+- `app` - should be set to the same value as `app.kubernetes.io/name`. It's
+  deprecated and only kept to avoid breaking existing workflows.
```

Comment:
We need to announce in the releases we do in order to give customers to adapt automation

Reply:
Sure, I'll mention that we need to do that in the ticket.